File: src/main/java/com/hjj/apiserver/service/DepositService.java
package com.hjj.apiserver.service;

import com.hjj.apiserver.dto.DepositDto;
import com.hjj.apiserver.repositroy.DepositRepository;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
@RequiredArgsConstructor
public class DepositService {

    private final DepositRepository depositRepository;

    public List<DepositDto.ResponseDepositFindAll> findDepositListByBankType(){

        return depositRepository.findDepositAllByBankType();
    }
}

File: src/main/kotlin/com/hjj/apiserver/service/DepositService.kt
package com.hjj.apiserver.service

import com.hjj.apiserver.dto.deposit.response.DepositFindAllResponse
import com.hjj.apiserver.repository.deposit.DepositRepository
import org.springframework.stereotype.Service

@Service
class DepositService(
    private val depositRepository: DepositRepository,
) {

    fun findDeposits():List<DepositFindAllResponse> {
        return depositRepository.findDepositAll()
    }
}